README.md

```diff
@@ -145,6 +145,88 @@ It will generate a very basic MSI file which installs the main executable and a
 
 Scryer Prolog must be built with **Rust 1.63 and up**.
 
+### Building WebAssembly
+
+Scryer Prolog has basic WebAssembly support. However, none of the default features are currently supported. You can manually change the default features in `Cargo.toml` from
```

Comment:
No need to edit Cargo.toml [`--no-default-features` command line flag](https://doc.rust-lang.org/cargo/reference/features.html#the-default-feature) or 
[`default-features = false`](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#choosing-features) in `[dependencies]` should be sufficient

Reply:
Should work with `wasm-pack build` as an [extra option](https://rustwasm.github.io/wasm-pack/book/commands/build.html#extra-options).

Reply:
> Should work with `wasm-pack build` as an [extra option](https://rustwasm.github.io/wasm-pack/book/commands/build.html#extra-options).

I agree that passing ` --no-default-features` to `wasm-pack` should theoretically work and is the recommended way. I even mentioned it in my other PR #1966.

However, it doesn't work on my PC (windows 10). I don't know why. `wasm-pack` seemed to ignore all extra parameters including `--offline`. I had to do a regular `cargo build --offline` and let `wasm-pack` "remember" I was using `--offline` last time. And I feel bad to write something in README.md that doesn't work for myself...

After some struggle I wrote down these silly instructions that at least should work for everyone. Anyway, writing the recommended way should be better. Do you think I should keep the workaround in case anyone else is facing the same problem, or remove it for now until I find the real cause on my side? I couldn't find any similar issue on github or stackoverflow so probably I'm the only one having this issue? 8-)

Reply:
@rujialiu: Personally, I think the instructions @Skgland posted seem preferable because they do not require any changes in existing files. If these instructions should work, but do not work for you, then could you please consider filing this as an issue in the [`wasm-pack`](https://github.com/rustwasm/wasm-pack) repository so that others will also benefit from the correction? Thank you a lot!

Reply:
Just to add: I think none of this is any reason to delay merging this PR, because we can easily tweak the instructions at any time! To me, it seems preferable to merge this work as soon as possible so that other PRs can already take it into account.

Reply:
REDAME.md updated again, thanks @Skgland and @triska for suggestions :) And I agree to merge this asap too!